nvflare/private/fed/server/server_engine.py

```diff
@@ -525,17 +525,20 @@ def register_aux_message_handler(self, topic: str, message_handle_func):
         self.run_manager.aux_runner.register_aux_message_handler(topic, message_handle_func)
 
     def send_aux_request(self, targets: [], topic: str, request: Shareable, timeout: float, fl_ctx: FLContext) -> dict:
-        if not targets:
-            self.sync_clients_from_main_process()
-            targets = []
-            for t in self.get_clients():
-                targets.append(t.name)
-        if targets:
-            return self.run_manager.aux_runner.send_aux_request(
-                targets=targets, topic=topic, request=request, timeout=timeout, fl_ctx=fl_ctx
-            )
-        else:
-            return {}
+        try:
+            if not targets:
+                self.sync_clients_from_main_process()
+                targets = []
+                for t in self.get_clients():
+                    targets.append(t.name)
+            if targets:
+                return self.run_manager.aux_runner.send_aux_request(
+                    targets=targets, topic=topic, request=request, timeout=timeout, fl_ctx=fl_ctx
+                )
+            else:
+                return {}
+        except:
+            self.logger.error(f"Failed to send the aux_message: {topic}")
```

Comment:
it might be good to have exception information here:

```
except Exception as e:
    self.logger.error(f"Failed to send the aux_message: {topic} with exception: {e}.")
```

Reply:
done.